Entity/CommentInterface.php

```diff
@@ -35,6 +37,14 @@ public function getThread(): ThreadInterface;
 
     public function setThread(ThreadInterface $thread): self;
 
+    public function getParent(): ?CommentInterface;
+
+    public function setParent(CommentInterface $parent = null): CommentInterface;
+
+    public function getDepth(): int;
+
+    public function getComments(): Collection;
```

Comment:
should we add the `getPublishedComments` method here too?

Reply:
👍